libs/langgraph/langgraph/prebuilt/chat_agent_executor.py

```diff
@@ -244,6 +240,7 @@ def create_react_agent(
     interrupt_before: Optional[list[str]] = None,
     interrupt_after: Optional[list[str]] = None,
     debug: bool = False,
+    **kwargs: dict[str, Any],
```

Comment:
should we call this something like "deprecated_kwargs" so people don't think they need to search here for real stuff?

Reply:
sure